Framework/Core/include/Framework/AnalysisManagers.h

```diff
@@ -29,6 +30,29 @@
 namespace o2::framework
 {
 
+template <typename ANY>
+struct GroupedCombinationManager {
+  template <typename TG, typename... T2s>
+  static void setGroupedCombination(ANY&, TG&, T2s&...)
+  {
+  }
+};
+
+template <typename T1, typename GroupingPolicy, typename H, typename G, typename... Us, typename... As>
+struct GroupedCombinationManager<GroupedCombinationsGenerator<T1, GroupingPolicy, H, G, pack<Us...>, As...>> {
+  template <typename TH, typename TG, typename... T2s>
+  static void setGroupedCombination(GroupedCombinationsGenerator<T1, GroupingPolicy, H, G, pack<Us...>, As...>& comb, TH& hashes, TG& grouping, std::tuple<T2s...>& associated)
+  {
+    static_assert(sizeof...(T2s) > 0, "There must be associated tables in process() for a correct pair");
+    static_assert(!soa::is_soa_iterator_t<std::decay_t<H>>::value, "Only full tables can be in process(), no grouping");
+    if constexpr (std::conjunction_v<std::is_same<G, TG>, std::is_same<H, TH>>) {
```

Comment:
why not &&?

Reply:
For passing the tables to the manager? But then the original tables (collisions, tracks) accessible inside process() would be invalidated (ownership loss), no? Even if there is no use case for using these tables in an event mixing task, this would be surely confusing to users. Similarly, in PartitionManager, we pass tables just by &.

Reply:
No, I meant why not using `std::is_same_v<G, TG> && std::is_same_v<H, TH>`. We can fix it later.